opengrok-indexer/src/main/java/org/opengrok/indexer/authorization/AuthorizationStack.java

```diff
@@ -219,7 +219,13 @@ protected boolean processStack(Nameable entity,
             PluginDecisionPredicate pluginPredicate,
             PluginSkippingPredicate skippingPredicate) {
 
-        boolean overallDecision = true;
+        Boolean overallDecision = null;
+        boolean optionalFailure = false;
+
+        if (getStack().isEmpty()) {
+            return true;
+        }
```

Comment:
"if" not necessary here, but i don't mind it

Reply:
I think it makes the code a little more easier to follow.